### ISSUE TITLE
Scrolling screenshot 

### ISSUE THREAD

**Issue by himanshugoswamiii** [naelstrof/maim#264]
I'm unable to find the scrolling screenshot feature (if it exists). 

If it doesn't, can it be implemented ??  (*Feature request*)

**Comment by bew** [comment]
What do you mean by "scrolling screenshot"?

**Comment by foxpy** [comment]
They probably mean an option to capture a window larger than output display. That sounds like a complicated feature to implement and I don't know anyone who could take it into development right now.

An interesting workaround is to use a compositing window manager (this covers basically 99% of all existing desktop setups), make a window larger than your output display and capture this window using `maim -s`.

**Comment by himanshugoswamiii** [comment]
@bew Take a look at Firefox screenshot feature.

we select an area, then we keep on scrolling and everything that comes into the area is included in our screenshot

**Comment by foxpy** [comment]
> @foxpy did you mean making window much smaller than the output display using `Ctrl -` and then taking the screenshot ??

No, I suggest making window so large it doesn't fit on your screen anymore and then taking a screenshot. Imagine having a 1920x1080 screen, resizing a window to something like 4000x3000 and then taking _a screenshot of this window_. If you have a compositing window manager (you almost certainly have one), the whole window will be screenshot'ed, despite being larger than your screen.

> Take a look at Firefox screenshot feature.

This one would be very difficult to implement. Firefox has direct control over web page while maim has basically zero control over X11 windows.

P.S. Sorry for a long answer, I am currently overwhelmed by life :)
